index.js

```diff
@@ -32,9 +28,18 @@ app.get('/todos', (request, response) => {
   });
 });
 
+app.get('/todos/:id',(request, response)=> {
+  connection.query('SELECT * FROM todos WHERE todoId = ?', [request.params.id], (error, data) => {
+    if (data[0] === undefined) {
+      return response.status(404).json({status: 404, data: error});
```

Comment:
Good job man, but I think instead of returning status of 404, maybe it's better to say no content.. with status `204` ..
Returning 404 signals that means the resource doesn't exist, kind of saying that it isn't even possible to ask for the list of albums for this particular id(todo). But in fact, it's possible to successfully return the todos of that todoId, it's just that the list is empty. 
While a 204 seems better than a 404, because it at least tells the client that the requrest was successful but had no content..